### DISCUSSION TITLE
如何在低代码中watch store中的全局变量

### DISCUSSION THREAD

**Discussion by stwflyfox** [opentiny/tiny-engine#915] [Q&A]
如题，如何深度监听全局store中的变量？

**Comment by chilingling** [comment]
在页面管理中，有生命周期配置。里面可以拿到 `watch`，然后通过 `stores.xxx` 拿到对应的全局状态。就可以进行监听了。

![image](https://github.com/user-attachments/assets/7f86aeb1-9de0-4e0b-b33a-082b61a98ec2)
![image](https://github.com/user-attachments/assets/fde8c5ff-1bf4-4619-9b06-7ca06cf58c17)



如果有问题欢迎反馈

**Comment by stwflyfox** [comment]
thanks
